internal/pkg/provision/providers/docker/network.go

```diff
@@ -16,7 +17,24 @@ import (
 	"github.com/talos-systems/talos/internal/pkg/provision"
 )
 
+// createNetwork will take a network request and check if a network with the same name + cidr exists.
+// If so, it simply returns without error and assumes we will re-use that network. Otherwise it will create a new one.
 func (p *provisioner) createNetwork(ctx context.Context, req provision.NetworkRequest) error {
+	existingNet, err := p.listNetworks(ctx, req.Name)
```

Comment:
should we update or remove the error message in https://github.com/talos-systems/talos/blob/master/internal/pkg/provision/providers/docker/create.go#L52-L54, as right now it will be triggered in the wrong case?

Reply:
yeah, I'm going to update that error message with PR #1716